src/screens/Login.js

```diff
@@ -0,0 +1,162 @@
+import React from "react";
+import {
+  View,
+  Text,
+  StyleSheet,
+  TextInput,
+  ImageBackground,
+  Image,
+  TouchableOpacity,
+} from "react-native";
+import AntDesign from "react-native-vector-icons/AntDesign";
+import Feather from "react-native-vector-icons/Feather";
+import { Colors } from "../colors/ConstantColors";
+
+const Login = () => {
+  return (
+    <View style={styles.container}>
+      <ImageBackground
+        style={styles.Img}
+        source={require("../../assets/1.jpg")}
```

Comment:
1. folder structure should be - `assests/images/imageName`
2. use appropriate name for images like appLogo.png etc